libs/blocks/icon-block/icon-block.js

```diff
@@ -75,6 +75,17 @@ function decorateContent(el) {
       if (secondColumn.children.length === 1) el.classList.add('items-center');
       el.querySelector('.foreground .text-content').append(secondColumn);
     }
+    const ctas = el.querySelectorAll('.action-area');
```

Comment:
These are div.action-area(s) right? Maybe update to `const actionAreas...`?  
Is there potential here for multiples. On the selectors in media.js and text.js you are only getting the single last-of-type. Im just trying to understand why this block may have multiple action areas where the others are assumed to have just one.

Reply:
Sure I can update the variable name.
Here's the use case:
<img width="1473" alt="Screenshot 2024-04-05 at 3 58 03 PM" src="https://github.com/adobecom/milo/assets/19690988/b9cdbba0-94ff-4935-90d7-63866243a532">
From https://main--milo--adobecom.hlx.page/drafts/methomas/text-alignment-all
It's kind of strange. In the icon block, we're adding an `action-area` class around the standalone link: https://github.com/adobecom/milo/blob/51a113452fccf258fdd91d6edfba348775a4ba25/libs/blocks/icon-block/icon-block.js#L63
So if there's a button and a standalone link, they're in separate action areas. Text and media blocks don't do this. If those blocks have links next to each other, they all get added to the same action area by the button decorator.